docs/intro.md

```diff
@@ -101,4 +101,7 @@ Slack is good for that, too: [join slack](https://greatexpectations.io/slack).
 If you have bugfix or feature request, please [upvote an existing issue](https://github.com/great-expectations/great_expectations/issues) or [open a new issue](https://github.com/great-expectations/great_expectations/issues/new) on GitHub and we’ll see what we can do.
 
 #### Contribute code or documentation
-We welcome contributions to Great Expectations. Please start with our [Contributing guide](/docs/contributing/contributing) and don’t be shy with questions!
\ No newline at end of file
+We welcome contributions to Great Expectations. Please start with our [Contributing guide](/docs/contributing/contributing) and don’t be shy with questions!
+
+#### Not interested in managing your own configuration or infrastructure? 
+Learn more about Great Expectations Cloud — our fully managed SaaS offering. Sign up for our weekly cloud workshop [here](https://greatexpectations.io/cloud). You’ll get to see our newest features and apply for our private Alpha program!
```

Comment:
```suggestion
Learn more about Great Expectations Cloud — our fully managed SaaS offering. Sign up for [our weekly cloud workshop](https://greatexpectations.io/cloud)! You’ll get to see our newest features and apply for our private Alpha program!
```

Reply:
In general it is preferred to have a link to a thing or action be the description of the thing or action, rather than having the thing or action be described and then providing a generic "here" link.